internal/client/policy/policy_resource.go

```diff
@@ -0,0 +1,263 @@
+
+	ManageV1alpha1WorkspacePolicyResourceServiceUpdate(request *policyworkspacemodel.VmwareTanzuManageV1alpha1WorkspacePolicyPolicyRequest) (*policyworkspacemodel.VmwareTanzuManageV1alpha1WorkspacePolicyPolicyResponse, error)
+}
+
+/*
+ManageV1alpha1ClusterPolicyResourceServiceCreate creates a policy.
+*/
+func (p *Client) ManageV1alpha1ClusterPolicyResourceServiceCreate(request *policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyPolicyRequest) (*policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyPolicyResponse, error) {
+	requestURL := fmt.Sprintf("%s/%s/%s", "v1alpha1/clusters", request.Policy.FullName.ClusterName, "policies")
+	policyClusterResponse := &policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyPolicyResponse{}
+	err := p.Create(requestURL, request, policyClusterResponse)
+
+	return policyClusterResponse, err
+}
+
+/*
+ManageV1alpha1ClusterPolicyResourceServiceDelete deletes a policy.
+*/
+func (p *Client) ManageV1alpha1ClusterPolicyResourceServiceDelete(fn *policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyFullName) error {
+	queryParams := url.Values{}
+
+	if fn.ManagementClusterName != "" {
+		queryParams["fullName.managementClusterName"] = []string{fn.ManagementClusterName}
+	}
+
+	if fn.ProvisionerName != "" {
+		queryParams["fullName.provisionerName"] = []string{fn.ProvisionerName}
+	}
+
+	requestURL := fmt.Sprintf("%s/%s/%s/%s?%s", "v1alpha1/clusters", fn.ClusterName, "policies", fn.Name, queryParams.Encode())
+
+	return p.Delete(requestURL)
+}
+
+/*
+ManageV1alpha1ClusterPolicyResourceServiceGet gets a policy.
+*/
+func (p *Client) ManageV1alpha1ClusterPolicyResourceServiceGet(fn *policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyFullName) (*policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyGetPolicyResponse, error) {
+	queryParams := url.Values{}
+
+	if fn.ManagementClusterName != "" {
+		queryParams["fullName.managementClusterName"] = []string{fn.ManagementClusterName}
+	}
+
+	if fn.ProvisionerName != "" {
+		queryParams["fullName.provisionerName"] = []string{fn.ProvisionerName}
+	}
+
+	requestURL := fmt.Sprintf("%s/%s/%s/%s?%s", "v1alpha1/clusters", fn.ClusterName, "policies", fn.Name, queryParams.Encode())
+	policyClusterResponse := &policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyGetPolicyResponse{}
+	err := p.Get(requestURL, policyClusterResponse)
+
+	return policyClusterResponse, err
+}
+
+/*
+ManageV1alpha1ClusterPolicyResourceServiceUpdate updates overwrite a policy.
+*/
+func (p *Client) ManageV1alpha1ClusterPolicyResourceServiceUpdate(request *policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyPolicyRequest) (*policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyPolicyResponse, error) {
+	requestURL := fmt.Sprintf("%s/%s/%s/%s", "v1alpha1/clusters", request.Policy.FullName.ClusterName, "policies", request.Policy.FullName.Name)
```

Comment:
nit : consider writing a helper function which will help you constructing these URL path separated by /